agent/command/results_utils.go

```diff
@@ -151,3 +152,15 @@ func makeCedarTestResults(id string, t *task.Task, results []testresult.TestResu
 
 	return rs, failed
 }
+
+// ToRedact returns the full list of expansion keys whose values
+// should get redacted from task logs.
+func ToRedact(redacted map[string]bool) []string {
```

Comment:
Can this get moved to `agent/internal/redactor`?

Reply:
This was one of the things that had been causing an import cycle.
I rearranged things so it wouldn't.